From f983a5cb441859aa944db52c3bdd5bc63772b8db Mon Sep 17 00:00:00 2001
From: Stefano Lattarini <stefano.lattarini@gmail.com>
Date: Thu, 21 Jun 2012 13:13:04 +0200
Subject: [PATCH] docs: avoid failures with development version of Texinfo
 (4.13.90)

* doc/automake.texi: Use '@item' instead of '@itemx' where
appropriate.

Signed-off-by: Stefano Lattarini <stefano.lattarini@gmail.com>
(backported from commit d0dc448517d3a61ba55e62dd76b9a42b68bef8f8)
Signed-off-by: Andreas Oberritter <obi@opendreambox.org>
---
 doc/automake.texi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/automake.texi b/doc/automake.texi
index c2d7edc..03a01a1 100644
--- a/doc/automake.texi
+++ b/doc/automake.texi
@@ -13039,7 +13039,7 @@ nicely.  Conditionals, Locations, Variable and Rule definitions,
 Options: these items on which Automake works have been rewritten as
 separate Perl modules, and documented.
 
-@itemx 2004-01-11 Automake 1.8.1
+@item 2004-01-11 Automake 1.8.1
 @itemx 2004-01-12 Automake 1.8.2
 @itemx 2004-03-07 Automake 1.8.3
 @itemx 2004-04-25 Automake 1.8.4
@@ -13056,13 +13056,13 @@ down to 1.2MB@.
 
 Aside from this it contains mainly minor changes and bug-fixes.
 
-@itemx 2004-08-11 Automake 1.9.1
+@item 2004-08-11 Automake 1.9.1
 @itemx 2004-09-19 Automake 1.9.2
 
 Automake has ten years.  This chapter of the manual was initially
 written for this occasion.
 
-@itemx 2007-10-29 Automake repository moves to @code{savannah.gnu.org} and uses
+@item 2007-10-29 Automake repository moves to @code{savannah.gnu.org} and uses
 git as primary repository.
 
 @end table
-- 
1.8.3.2

